hack/stabilization-changes.py

```diff
@@ -474,10 +474,10 @@ def notify(message, webhook=None):
 
 
 def promote(version, channel_name, channel_path, subject, body, upstream_github_repo, push_github_repo, github_token, upstream_branch, labels=None):
+    upstream_remote = get_remote(repo=upstream_github_repo)
+    branch = 'promote-{}-to-{}'.format(version, channel_name)
```

Comment:
Hmm, maybe we should drop `in {}/{}` from the `has already been promoted to` error message below instead?  Because it looks like the check is just "that version is already in that channel", and doesn't inspect to decide how the version got into the channel.  Then we could leave these variables under the `if github_token` branch, because there would be no later consumers.

Reply:
We could, but  there are `branch` consumers later though. They are protected by the `if not github_token: ... return pull` guard. I dislike semantic code dependencies like that but I guess I can live with that :)